_docs/organizations-and-collections.md

```diff
@@ -0,0 +1,92 @@
+
+There are two types of roles available:
+* **Maintainer** - can update organization, collections and membership
+* **Member** - can only update organization and collections
+
+Membership can be updated on the membership tab of the organization page. A maintainer cannot delete their own membership.
+
+When a user is requested to join an organization, they will get an invite which is available at the `requests` tab of the [accounts](https://dockstore.org/accounts) page. Here they can either accept or reject the invitation.
+
+## Collections
+Collections are a way of gathering related tools and workflows in an easily accessible location. They can be used for grouping tools and workflows for a specific grant, theme, field, etc. A collection is only publicly visible if the organization that it belongs to is approved.
+
+### Creating a collection
+To create a collection, go to the collections tab on the organization page and select `Create collection`.
+![Create Collection](/assets/images/docs/CreateCollection.png)
+
+Collections have the same core information as organizations. The name, display name, and topic are all **required**. They can be changed later.
+* **Name** - the name used in URLs and as an identifier (unique within organization)
+* **Display Name** - the pretty name used anywhere the collection is mentioned (unique within organization)
+* **Topic** - a short description of the collection (1-2 sentences)
+
+**Note**: Tools and workflows are added to the collection after it is created.
+
+![Collection](/assets/images/docs/CollectionView.png)
+
+### Updating the metadata
+All of the information that was defined in the add collection form can be updated after the collection is created.
+
+A freeform markdown description can be added to a collection, however it is entirely optional. It is recommended that collections have at least a basic description.
+
+### Adding tools and workflows
+Only published tools and workflows can be added to a collection. If a tool belonging to a collection is unpublished, it will be hidden until it is published again. To add a tool or a workflow to a collection, go to the public page for the entry and click `Add to collection` on the right-hand side.
+
+![Add to Collection](/assets/images/docs/AddToCollection.png)
+
+This will open a dialog where you can select which collection to add the entry to.
+![Add to Collection Dialog](/assets/images/docs/AddToCollectionModal.png)
+
+Once added, the entry will appear on the collection page. If a user wants to remove an entry from a collection, they can do so from the collections page.
+
+The `Current Collections` seciton will now link to the PCAWG collection.
+![Add to Collection PCAWG](/assets/images/docs/CurrentCollectionsWithPCAWG.png)
+
+## Events
+We keep track of events occurring related to organizations and collections and display the important ones in the `events` tab of the organization. Only a subset of all events are shown.
```

Comment:
> Only a subset of all events are shown

Leaves me wondering what the subset is. I would either remove this sentence entirely, or expand to say what events are shown.